lambda_fns/extract_docs/app.py

```diff
@@ -68,6 +76,39 @@ def store_text_s3(data, bucket_name, key):
         Key=key
     )
 
+def upload_file_to_s3(
+    url,
+    key="temporaryfile.pdf",
+    bucketname="deep-large-docs-conversion"
```

Comment:
Let's not define the default value here or use the value from config.

Reply:
done.